README.md

```diff
@@ -152,31 +167,85 @@ These steps are relevant for javascript lambda functions, and assume you have a
     }
     ```
 
+    Your `tasks.json` file should now look similar to this:
+    
+    ```json
+    {
+        "version": "2.0.0",
+        "tasks": [
+
+            <In here you may have 0 or more additional task entries>
+
+            {
+                "label": "launchLambdaFunction",
+                .
```

Comment:
I've never seen an ellipsis written vertically like this--I think it would be confusing to readers.